src/Makefile.manual

```diff
@@ -19,6 +19,7 @@ SRCFYPP =\
         stdlib_stats_moment_mask.fypp \
         stdlib_stats_moment_scalar.fypp \
         stdlib_stats_var.fypp \
+		stdlib_math.fypp \
```

Comment:
```suggestion
	stdlib_math.fypp \
```
```suggestion
        stdlib_stats_var.fypp \
        stdlib_math.fypp \
```

Reply:
Noted! I was using 2 tabs to indent, and on my VS code it was aligned properly but when I push it to GitHub I see indentation differences. So I replaced 2 tabs with 8 spaces to resolve this issue and it worked.

Reply:
In general you should avoid using tabs `'\t'` for indentation in Fortran. I think some compilers allow it, but not all of them. You should set your editor to replace tabs with spaces (personally I use 2 spaces, but this will be project dependent).